content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
++++
+title = "Kubeflow on-prem in k8s multi-node cluster"
+description = "How to install Kubeflow on-prem leveraging Dynamic Volume Provisioning and NFS volumes"
+weight = 30
++++
+
+This guide describes how to setup Kubeflow on premise in a multi-node cluster using dynamic volume provisioning.
+
+## Vanilla on-prem KubeFlow installation
+
+In order to install KubeFlow in a on-prem Kubernetes cluster (both single node and multi-node) you can follow this guide
+[Kubeflow on Kubernetes](/docs/started/getting-started-k8s).
+
+At the end of the installation, as stated in the troubleshooting section [Pods stuck in Pending state](/docs/other-guides/troubleshooting/#pods-stuck-in-pending-state), you could have some Persistent Volume Claims (PVCs) ubounded.
+
+In fact, in KubeFlow there are three pods that have 10Gi PVCs that will get stuck in pending state 
+if they are unable to bind their PVC. The three pods are minio, mysql, and vizier-db. 
+You can check the status of the PVC requests:
+
+```
+kubectl -n ${NAMESPACE} get pvc
+```
+
+  * Look for the status of "Bound"
+  * PVC requests in "Pending" state indicate that the scheduler was unable to bind the required PVC.
+
+In order to change status of PersistentVolumeClaims in "Bound" three Persistent Volumes (PVs) must be provisioned.
+
+But wait! 
+
+There are a couple of problems to solve before: you are in a multi-node cluster, you cannot simply create an *hostPath* PersistentVolume (the volume that exposes a filesystem directory to a Pod) like in a single node scenario.
+
+## A step back: the storage on Kubernetes
+
+Kubernetes defines several ways to attach Volumes to Pods. 
+
+The best practice consists in decoupling the need for storage ([Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims): PVC) from the real storage ([Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/): PV). The latter is provisioned by Kubernetes administratore based on the available resources. PV can be one of the [several supported](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).
+
+Cloud providers define, in their Kubernetes cluster, mechanisms to allocate PVs based on existing PVCs using their storage infrastructure; on-prem cluster must provision their PV according the existing capability of the system.
+
+In development clusters (e.g. Minikube) or single node clusters, PVCs can be binded to HostPath PVs: a particular kind of volume that maps the Pod's Volume to a directory of the filesystem.
```

Comment:
Earlier in the document you use the capitalization *hostPath* (in italics). I don't know which one is more correct (HostPath vs. hostPath - italics? code font?). I trust your expertise, but can you make sure it is consistent throughout the document?

Reply:
I like more HostPath. For the moment I'll use this version uniformly and I'll wait for the expert advice to change it